src/main.js

```diff
@@ -192,9 +192,19 @@ function showMyPoster() {
   mainPoster.style.display = "block";
   posterForm.style.display = "none";
 }
+function testForDuplicate(arrayOfPosters, mainPoster) {
+  for(var i = 0; i < arrayOfPosters.length; i++) {
+    if (arrayOfPosters[i].imageURL === mainPoster.imageURL &&
+        arrayOfPosters[i].title === mainPoster.title &&
+        arrayOfPosters[i].quote === mainPoster.quote) {
+          return true;
+        }
```

Comment:
I tend to not really like multi-line if statements. What if this particular bit of logic was further broken out into its own function? What would something like that look like? 

I imagine something like the following: 

``` 
if ( determinePosterMatch(mainPoster, arrayOfPosters[i]) ) {
  return true
}
```